mpt/mavenplugin/pom.xml

```diff
@@ -208,4 +200,13 @@ to the library requiring no extra coding.</description>
             </build>
         </profile>
     </profiles>
+
+    <reporting>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-plugin-plugin</artifactId>
```

Comment:
are you sure it does the combine.children="append" ?

Reply:
I have no idea of how to merge reports plugin together.

https://maven.apache.org/guides/mini/guide-configuring-plugins.html#Configuring_Reporting_Plugins seems to suggest inheritance is the default. I would not worry about this.